### PR TITLE
Helper class for 3d-tiles tests

### PR DESCRIPTION
**Pull request by lilleyse**
For #3177 

I added a helper class called `Cesium3DTilesSpecHelper` which has most of the common functions used in the tile tests.

### CODE REVIEW

File: Specs/Cesium3DTilesSpecHelper.js [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+        Color,
+        defaultValue,
+        OrientedBoundingBox,
+        Cesium3DTileContentProviderFactory,
+        Cesium3DTileContentState,
+        Cesium3DTileset,
+        pollToPromise) {
+    "use strict";
+
+    var Cesium3DTilesSpecHelper = function() {
```
Comment (pjcozzi) [inline Specs/Cesium3DTilesSpecHelper.js]: We avoid using `Helper` in names.  Do you have a better name?  I dunno, `Cesium3DTilesTester`.

File: Specs/Cesium3DTilesSpecHelper.js [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+    }
+
+    function verifyRenderBlank(scene, tileset) {
+        tileset.show = false;
+        expect(scene.renderForSpecs()).toEqual([0, 0, 0, 255]);
+        tileset.show = true;
+        expect(scene.renderForSpecs()).toEqual([0, 0, 0, 255]);
+    }
+
+    Cesium3DTilesSpecHelper.verifyRenderTileset = function(scene, tileset) {
```
Comment (pjcozzi) [inline Specs/Cesium3DTilesSpecHelper.js]: Perhaps we should rename these functions with calls to `expect` to something like `expectRenderTileset ` (I know `verify` is used in old tests).

**Comment by pjcozzi** [comment]
This looks good, just those naming comments.

**Comment by lilleyse** [comment]
Updated.

**Comment by pjcozzi** [comment]
Looks good.

This can be a separate PR, but in this branch (and `3d-tiles`), this test fails on my MacBook Pro:

image

**Comment by lilleyse** [comment]
Okay I'll look into that.

**Pull request merged**